Tools/visualize_log.py

```
@@ -188,18 +140,18 @@ def plot_telemetry(telemetry_df: pd.DataFrame, event_df: pd.DataFrame) -> None:
-        telemetry_file_path = _find_latest_telemetry_file()
-        event_file_path = _find_latest_event_log()
-    if not telemetry_file_path or not event_file_path:
+    if FLAGS.telemetry_file and FLAGS.event_log:
+        telemetry_file_path = Path(FLAGS.telemetry_file)
+        event_log_path = Path(FLAGS.event_log)
+    else:
+        telemetry_file_path = utils.find_latest_telemetry_file(
+            FLAGS.log_search_dir)
+        event_log_path = utils.find_latest_event_log(FLAGS.log_search_dir)
+    if not telemetry_file_path or not event_log_path:
         raise ValueError("Missing required telemetry or event log file.")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)